AllReadyApp/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: "3"
+services:
+    web:
+        build: 
+            context: .
+            dockerfile: Web-App/AllReady/Dockerfile
+        container_name: allreadyweb
+        environment:
+            ASPNETCORE_ENVIRONMENT: Development
+            Data__DefaultConnection__ConnectionString: "Server=db,1433;Initial Catalog=master;Persist Security Info=False;User ID=sa;Password=yourStrong(!)Password;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=True;Connection Timeout=30;"
+            Data__HangfireConnection__ConnectionString: "Server=db,1433;Initial Catalog=master;Persist Security Info=False;User ID=sa;Password=yourStrong(!)Password;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=True;Connection Timeout=30;"
+        ports:
+            - "8000:80"
+        depends_on:
+            - db
```

Comment:
Since the db container takes a bit of time to spin up, I'd suggest adding
```yaml
restart: on-failure
```
So that docker could retry starting the app if the db is not yet up and running.